pygount/analysis.py

```diff
@@ -591,29 +594,38 @@ def _paths_and_group_to_analyze(self, path_to_analyse_pattern, group=None) -> It
                         yield path_to_analyse, actual_group
 
     def _source_paths_and_groups_to_analyze(self, source_patterns_to_analyze) -> List[Tuple[str, str]]:
+        def _process_source_pattern(source_pattern):
+            remote_url, revision = git_remote_url_and_revision_if_any(source_pattern)
+            if remote_url is not None:
+                git_storage = GitStorage(remote_url, revision)
+                self._git_storages.append(git_storage)
+                git_storage.extract()
+                # TODO#113: Find a way to exclude the ugly temp folder from the source path.
+                result.extend(self._paths_and_group_to_analyze(git_storage.temp_folder))
+            else:
+                has_url_prefix = re.match(HAS_URL_PREFIX, source_pattern)
```

Comment:
Minor inconsistency: The result of `result.match()` is a `Match`object, not a boolean. To be more precise:
```python
has_url_prefix = re.match(HAS_URL_PREFIX, source_pattern) is not None
```